capture_gui/lib.py

```diff
@@ -194,7 +195,7 @@ def _browse(path):
             default_filename = "playblast"
 
         # Default to images rule (if rule exists in workspace)
-        images_rule = mc.workspace("images", q=1, fre=True)
+        images_rule = pm.workspace.fileRules["images"]
```

Comment:
What's the reason for swapping to Pymel? Is this impossible to perform safely with maya.cmds? It's not a big problem but Pymel (on first load) does introduce some overhead and I was wondering whether and how this could be avoided if need be.